plugins/processors/t128_filter/README.md

```diff
@@ -18,6 +18,12 @@ The `t128_filter` filters out metrics passing through it. This processor is usef
     ##  * "regex": go flavored regex comparison
```

Comment:
I'm not going to do this rn, but I feel like most of this readme could be dynamically filled in by the code. Like this sample configuration is exactly the same as the telegraf sample configuration. The examples could be derived from testcases. Then nothing would get out of sync and wouldn't have to keep two places up to date.

Reply:
sounds like you're describing [rust](https://doc.rust-lang.org/rustdoc/documentation-tests.html) 🙂

Reply:
Oh man, that looks sweet